test/integration/addons_test.go

```diff
@@ -46,19 +47,45 @@ func testAddons(t *testing.T) {
 	}
 }
 
+func readLineWithTimeout(b *bufio.Reader, timeout time.Duration) (string, error) {
```

Comment:
Should we merge/refactor this with `readByteWithTimeout` from https://github.com/kubernetes/minikube/pull/3322?

Reply:
A good point, but this function is both different and trivial enough that de-duping it doesn't seem helpful. In this case, we're reading from minikube, where we know that either a newline will be output with useful output, or a timeout will occur, in which it's an error.

Reply:
A bad habit. I've decided in this case to just remove the log statements, since they are not actually all that helpful. When an error occurs, it should now be self-evident where the issue lies.